app/controllers/analytics/facilities_controller.rb

```diff
@@ -22,7 +22,14 @@ def set_facility
   end
 
   def set_facility_group
-    @facility_group = @facility.facility_group
+    if FeatureToggle.enabled?('DASHBOARD_VIEW_BY_DISTRICT')
+      organization = Organization.find(FacilityGroup.find(@facility.facility_group_id).organization_id)
```

Comment:
Move this to a seperate function?
Also, we have set_organization. Can we create the district using 
```
OrganizationDistrict.new(@facility.district, @organization)
```

Reply:
Done - updated creation of `OrganizationDistrict`, created a new function for organization district.